bedrock/mozorg/templates/mozorg/about/leadership/senior.html

```diff
@@ -478,5 +480,23 @@ <h2 class="group-title">Senior Leadership</h2>
     </div>
   </article>
 
+  <article id="ian-carmichael" data-id="ian-carmichael" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
```

Comment:
If there's no bio yet, we can remove `has-bio` to avoid the modal showing empty content when clicked?

It looks like the list of people should also be ordered alphabetically by surname, so Ian should come third, after Katherine Bose?